src/app/kontakt/components/InfoList.tsx

```diff
@@ -0,0 +1,12 @@
+import { PropsWithChildren } from "react";
+
+interface InfoListProps {
+  className?: string;
+}
+
+export const InfoList = ({
+  children,
+  className,
+}: PropsWithChildren<InfoListProps>) => (
+  <div className={`space-y-10 ${className ?? ""}`}>{children}</div>
```

Comment:
You can use our classnames util for joining classes and using conditional classes.

Also if it is a list then please use proper HTML tag for it. Same for the items - you can wrap the items in `<li />` tag